app/views/settings/category/_email.html.erb

```diff
@@ -1,3 +1,3 @@
-<%= link_to_function(_("Test email"), "test_mail(this, #{User.current.id}, '#{test_mail_user_url(User.current)}')",
+<%= link_to_function(_("Test email"), "tfm.users.testMail(this, '#{test_mail_user_url(User.current)}')",
```

Comment:
We don't actually need to send an ID in this case, the ID is already being sent from the url path.